src/training/stylegan_adversarial.py

```diff
@@ -205,9 +207,9 @@ def validate(self, epoch: int):
             cycle = self.model_G(style_label_embeds, reconstructed_style_embeds)
             cycle_loss = self.cons_loss(style_imgs, cycle)
 
-            perc_loss, tex_loss = self.perc_loss(style_imgs, res)
-            emb_loss = self.typeface_loss(style_imgs, res)
-            adv_loss = self.model_G_adv_loss(res)
+            perc_loss, tex_loss = self.perc_loss(style_imgs, preds)
+            emb_loss = self.typeface_loss(style_imgs, preds)
+            adv_loss = self.model_G_adv_loss(preds)
```

Comment:
dont we want to calculate g loss with reconstructed images?

Reply:
yes u r right, my bad